daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -758,7 +760,12 @@ abstract class TestCase(testCaseXML: NodeSeq, val parent: DFDLTestSuite) {
       case (None, Some(uri)) => {
         //
         // In this case, we have a real TDML file (or resource) to open
-        URISchemaSource(uri)
+        val path = if (Misc.isNullOrBlank(uri.getPath)) {
```

Comment:
Is this in case uri is a "jar" URI? Maybe this goes away with the `uriToDiagnosticPath` idea mentioend above?

This probably raises an issue that we want to be careful about calling uri.getPath, since that doesn't work for "jar" uris. I imagine we don't do that except for converting URI's to diagnostics.